public/assets/css/mapDesktop.css

```diff
@@ -1,10 +1,10 @@
 @media only screen and (min-width: 1101px) {
   .home {
     width: 100%;
-    height: 100vh;
-    overflow: auto;
+    overflow: hidden;
     display: flex;
     flex-direction: row;
+    max-height: calc(100vh - var(--height-header) - var(--height-footer));
```

Comment:
since you are using this calculation method in multiples elements / files, maybe you could create a variable for it inside global css file.
With this method, if one day you have to add another element in the calculation, you'll just have to change one line in global css file